src/TSHelper.ts

```diff
@@ -157,6 +164,13 @@ export function isExplicitArrayType(type: ts.Type, checker: ts.TypeChecker, prog
         return true;
     }
 
+    if (type.symbol) {
```

Comment:
It shouldn't affect correctness, but would it maybe be better to make it a first check in the function, so that a technical detail is checked before the actual function logic?